content/cran-mirror/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Mirror of CRAN packages"
```

Comment:
I think we need to change the title to reflect that this is a different kind of mirror, and the goal is not to provide a download service, but to be able to browse and search the source code.

Reply:
"Searchable copy"?

Reply:
I don't mind the work "mirror", because that is used already, how about sg like "CRAN source code mirror"?